aztec_macros/src/utils/hir_utils.rs

```diff
@@ -245,8 +247,10 @@ pub fn inject_global(
 
     let def_maps = &mut context.def_maps;
 
-    let path_resolver =
-        StandardPathResolver::new(ModuleId { local_id: module_id, krate: *crate_id });
+    let module_id = ModuleId { local_id: module_id, krate: *crate_id };
+    let parent_module_id = context.def_interner.try_module_parent(&module_id);
+
+    let path_resolver = StandardPathResolver::new(module_id, parent_module_id);
```

Comment:
`parent_module_id` should be accessible through the `def_maps` argument which is passed to `resolve`. (it's in the `ModuleData` field).

Reply:
Oh, you are right! I didn't realize `Arena` was a `Vec` indexed by whatever can be converted to `Index`.

I'll update this PR and might also remove `parent_id` from somewhere else (I needed it for the lsp hover).

Reply:
Done, thank you!